capstone/src/main/webapp/scripts/post.js

```diff
@@ -1,27 +1,85 @@
 function init() {
-  fetchBlobstoreUrlAndShowForm();
   loadPosts();
   getPollOptions();
-
+  uploadImage();
+  //fetchBlobstoreUrlAndShowForm();
```

Comment:
this comment can be deleted if you don't need it anymore

Reply:
oops not sure if I reviewed it correctly but I approved the changes in the sense that the stuff I commented on yesterday seems resolved!

Reply:
removed